torch/ao/quantization/pt2e/eval_utils.py

```diff
@@ -21,26 +21,28 @@ def _replace_dropout_for_eval(m: torch.fx.GraphModule):
     m.graph.eliminate_dead_code()
     m.recompile()
 
-    def dropout_train(x):
-        return F.dropout(x, p=0.5, training=True)
+    for inplace in [False, True]:
 
-    def dropout_eval(x):
-        return F.dropout(x, p=0.5, training=False)
+        def dropout_train(x):
+            return F.dropout(x, p=0.5, training=True, inplace=inplace)
```

Comment:
can we use ignore_literals just for `inplace`?

Reply:
Looks like `ignore_literals` already enabled in `replace_pattern_with_filters`, but it still failed to match inplace dropout per the result of UT in this PR.
https://github.com/pytorch/pytorch/blob/34ea0a2bdc7d17f5ca0e895590159e5fd36fffa0/torch/ao/quantization/pt2e/eval_utils.py#L36-L43

Reply:
@jerryzh168 Good catch. I think this is actually wrong right now. Since we do `ignore_literals=True` we actually won't match `inplace` correctly.

@leslie-fang-intel It seems we need a match filter to make sure `inplace` is actually the value we expect. Can you add that in a follow up PR? I also don't understand how the existing test passed.

Reply:
> @leslie-fang-intel It seems we need a match filter to make sure inplace is actually the value we expect. Can you add that in a follow up PR? I also don't understand how the existing test passed.

Since there is actually no `inplace` parameter for the dropout aten operator, not sure whether the `match filter` is still needed here. @andrewor14

Reply:
I see, it's because the decomposed aten ops don't have the `inplace` flag, so matching still works. Thanks for explaining.